test/unit/fixture/search_full_address.js

```diff
@@ -1,6 +1,4 @@
-
-var peliasQuery = require('pelias-query'),
-    vs = new peliasQuery.Vars( peliasQuery.defaults );
+var vs = require('../../../query/search_defaults');
```

Comment:
should be using search specific defaults, instead of common pelias defaults (see in other search fixtures as well)